api/geth_backend.go

```diff
@@ -791,10 +763,29 @@ func (b *GethStatusBackend) startNodeWithAccount(acc multiaccounts.Account, pass
 		return err
 	}
 
-	err = b.SelectAccount(login)
-	if err != nil {
-		return err
+	if chatKey == nil {
```

Comment:
@igor-sirotin since this is the same snippet as one started in line 643, could we move it to a new function?

Reply:
I gave up on this, because `startNodeWithAccount` is more or less a copy of `loginAccount` (the duplicate you pointer is in it). 
So I just deprecated `startNodeWithAccount`. But it's still used in pretty many places, so I didn't bother yet to remove it. Opened an issue to track this: https://github.com/status-im/status-go/issues/5364